src/Interpreters/getOSKernelVersion.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+#if defined(OS_LINUX)
+
+#include <Common/typeid_cast.h>
+
+#include <string>
+#include <sys/utsname.h>
+
+namespace DB
+{
+
+/// Returns String with OS Kernel version.
+/* To get name and information about current kernel.
+   For simplicity, the function can be implemented only for Linux. 
+*/
+    
+String getOSKernelVersion();
```

Comment:
BTW right now it is not used anywhere, so at least logging it's result during startup should be added I guess (although for now it does not differs from `Poco::Environment::OS*` that is already printed at startup, see previous comment)

Reply:
Makes sense. What do you think? @alexey-milovidov

Reply:
Makes sense. What do you think? @alexey-milovidov